catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -89,7 +89,7 @@ const ImageEndpointSearchResults: FunctionComponent<Props> = ({
       <ImageContainer key={galleryImage.key} role="listitem">
         <ImageCard
           id={galleryImage.photo.id}
-          workId={galleryImage.photo.workId}
+          workId={galleryImage.photo.source.id}
```

Comment:
Looks fine; is there a way we could have leant on the type system to catch this for us? Do we know what the type of `galleryImage` is?

Reply:
`galleryImage` it's what is passed in when using `renderImage={(our function to render images)}` with `react-photo-gallery` and they didn't offer a type for it. You can kind of customise it too (in our case, whatever is in "photo" is ours), but I'd say it'd be hard to type since it comes from an external lib? What do you think, should it be attempted?